### ISSUE TITLE
Saving and loading a fine tuned cascade mask rcnn model

### ISSUE THREAD

**Issue by FirasKedidi** [open-mmlab/mmdetection#5232]
Hello,
I used mmdetection to fine-tune a cascade mask rcnn model on a custom dataset.
I saved the trained model in pth.
But when I wanted to use abroad and in another environment I faced problems.
I saved it using torch.save(model.static_dict(),path).
When I loaded I need to create an object model from the Model Class.
How can I do that using the saved model ??
Thanks

![image](https://user-images.githubusercontent.com/58661767/119697730-6b2d8480-be48-11eb-9357-a06ae6061e36.png)

**Comment by FirasKedidi** [comment]
If the saving technique I've used is wrong, please show me how to save it correctly.
Thanks

**Comment by RangiLyu** [comment]
The static_dict of a model only has param names and weights, it does not have the model graph. It is necessary to create a model object before loading weights.
